tests/test_hermes_doctor_bank_routing.py

```
@@ -0,0 +1,361 @@
+from pathlib import Path
+
+import pytest
+
+from mnemosyne_hermes.cli import _resolve_cli_bank
+
+
+# ---------------------------------------------------------------------------
+# Helpers
+# ---------------------------------------------------------------------------
+
+def _args(**kw):
```

Ruff (0.15.20)

[warning] Missing return type annotation for private function `_args`

(ANN202)